<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogIn</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_tf">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>270</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Enter your username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_tf">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter your password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_in_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flag_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
